src/Stryker.Core/Stryker.Core/MutationTest/MutationTestProcess.cs

```diff
@@ -138,6 +138,18 @@ public void Mutate(StrykerOptions options)
         public StrykerRunResult Test(StrykerOptions options)
         {
             var mutantsNotRun = _input.ProjectInfo.ProjectContents.Mutants.Where(x => x.ResultStatus == MutantStatus.NotRun).ToList();
+            if (!mutantsNotRun.Any())
+            {
+                if (_input.ProjectInfo.ProjectContents.Mutants.Any(x => x.ResultStatus == MutantStatus.Skipped))
```

Comment:
I think we can just use Any here? If there are > 0 mutants but no mutants to run, it means the other mutants were excluded.

```suggestion
                if (_input.ProjectInfo.ProjectContents.Mutants.Any()
```

Reply:
Well other mutations were excluded, that's correct but what if the other mutations did not compile?
Then we would say that all mutations were excluded, is this expected behavior? Since user is not able to re-run with less exclusion, compile error mutations.

Current logic(any => skipped): 

|mutations to run | skipped | compile error| message|
|-------------------|----------|----------------|----------|
|0|0|0|It's a mutant-free world, nothing to test.|
|0|1|0|It looks like all mutants were excluded, try a re-run with less exclusion.|
|0|0|1|It's a mutant-free world, nothing to test.|

Current logic(any): 

|mutations to run | skipped | compile error| message|
|-------------------|----------|----------------|----------|
|0|0|0|It's a mutant-free world, nothing to test.|
|0|1|0|It looks like all mutants were excluded, try a re-run with less exclusion.|
|0|0|1|It looks like all mutants were excluded, try a re-run with less exclusion.|